opentreemap/api/auth.py

```diff
@@ -68,18 +78,30 @@ def firstmatch(regx, strg):
 
 
 def decodebasicauth(strg):
```

Comment:
Now that have moved the actual decoding into `parse_basicauth` consider renaming this to `split_basicauth` or similar.